tests/app/questionnaire/conftest.py

```diff
@@ -815,6 +818,194 @@ def section_with_repeating_list():
     }
 
 
+@pytest.fixture
+def section_with_labels():
```

Comment:
The fixture name isn't very clear. Maybe just `labels_schema_with_placeholders`  would be clearer.